src/tests/user.test.ts

```diff
@@ -20,7 +22,28 @@ afterEach(async () => {
 describe('Testing User Endpoints', () => {
   describe('[GET] /users', () => {
     it('successfully get all users', async () => {
-      return await request(app.getServer()).get(`${userRoute.path}`).set('Cookie', [testCookie]).expect(200);
+      return await request(app.getServer()).get(`${userRoute.path}`).set('Cookie', [testAccessCookie]).expect(200);
+    });
+  });
+  describe('[GET] /users/:id', () => {
+    const userData: BaseUserResponseDTO = {
+      id: 1,
+      firstName: 'firstName',
+      lastName: 'lastName',
+      email: 'test@email.com',
+      language: 'chinese',
+      dateOfBirth: null,
+      profilePic: null,
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    };
+    it('successfully gets a specific user', async () => {
+      UserEntity.findOne = jest.fn().mockImplementation(() => userData);
+      return await request(app.getServer()).get(`${userRoute.path}/:id`).set('Cookie', [testAccessCookie]).send(String(userData.id)).expect(200);
+    });
+    it('returns 409 status code if user is not found', async () => {
```

Comment:
It might be worth creating a shared utility function that sets a test cookie and sends the request. 

something like 
const testAccessCookie = authService.createTestCookie('access', 'access');
const sendTestRequestWithCookie = async ({app: Express.Application, path: string, body?:any, expectedStatusCode: number }) => {
  return await request(app.getServer()).get(path).set('Cookie', [testAccessCookie]).send(body).expect(expectedStatusCode);
}

Reply:
Also you do know that `.get(`${userRoute.path}/:id`)` and 
`.send(String(userData.id))` is technically incorrect and will do nothing but will pass the test because we're mocking the methods right?